spec/support/helpers/authentication_helper.rb

```diff
@@ -0,0 +1,8 @@
+module AuthenticationHelper
+  def sign_in_as(user)
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.